velox/connectors/hive/storage_adapters/hdfs/CMakeLists.txt

```diff
@@ -19,8 +19,18 @@ add_library(velox_hdfs RegisterHdfsFileSystem.cpp)
 if(VELOX_ENABLE_HDFS)
   target_sources(velox_hdfs PRIVATE HdfsFileSystem.cpp HdfsReadFile.cpp
                                     HdfsWriteFile.cpp)
-  target_link_libraries(velox_hdfs Folly::folly ${LIBHDFS3} xsimd)
+  target_link_libraries(velox_hdfs Folly::folly ${LIBHDFS} ${LIBJVM} xsimd
+                        arrow)
+  target_include_directories(velox_hdfs PRIVATE ${HADOOP_HOME}/include/)
+  if(${VELOX_BUILD_TESTING})
+    add_subdirectory(tests)
```

Comment:
Do we may add_subdirectory(tests) twice when both VELOX_ENABLE_HDFS and VELOX_ENABLE_HDFS3 is ON